dask/array/tests/test_slicing.py

```diff
@@ -378,3 +379,61 @@ def test_slice_list_then_None():
     y = x[[2, 1]][None]
 
     assert eq(y, np.zeros((1, 2, 5)))
+
+
+class ReturnItem(object):
+    def __getitem__(self, key):
+        return key
+
+
+@skip
```

Comment:
possibly a better idea to use something like `unittest.skip` here? that way it gets listed as a skipped test and it's harder to forget about it :)